tasks/selinux.py

```diff
@@ -0,0 +1,37 @@
+"""
+SELinux namespaced tasks
+"""
+import os
+
+import invoke
+from invoke import task
+
+DEFAULT_SYSPROBE_POLICY_TE_FILE = os.path.join(".", "cmd", "agent", "selinux", "system_probe_policy.te")
```

Comment:
the command might be more resilient if we make these paths relative to the `selinux.py` itself?
```
DEFAULT_SYSPROBE_POLICY_TE_FILE = os.path.join(os.path.realpath(__file__), "..", "cmd", "agent", "selinux", "system_probe_policy.te")
```

Reply:
The complete command will be a bit more complicated (you can't append `..` to `os.path.realpath(__file__)` because it's a file, not a directory), but yeah that's doable.

Reply:
ah yeah, my bad.... you know what I mean ;)